salt/profiles/config/etc-syslog-ng-conf.d-profiles.conf

```diff
@@ -0,0 +1,16 @@
+@version: 3.5
+
+source s_profiles_application_all {
+    file("/srv/profiles/var/logs/*.json"
```

Comment:
`.log`

Reply:
Do we need to worry about extensions, or can we just rotate anything in the `logs` directory? (`/srv/profiles/var/logs` could just be in pillar then.)

Reply:
Changed to target only the JSON one to avoid duplicates in Loggly and such

Reply:
https://github.com/elifesciences/profiles/pull/25/commits/94ffb9750007a65288084e9f580c56ec01aa3f29

Reply:
Ah ok. Still, `/srv/profiles/var/logs` could be in pillar.

Reply:
We can rotate anything with the extension because after rotation the extension is changed; some like `peerscout` make folder paths customizable but the log directory is already customizable at the application level by configuring something else in `app.cfg`.
But in the formula, which is used only in builder-based environments, I wouldn't want servers in different environments or local virtual machines to use a different path; the only use case we have found so far has been putting a folder on a different volume, but it is usually solved with a symlink to keep consistency.